themes/default/content/blog/iac-recommended-practices-structuring-pulumi-projects/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Iac Recommended Practices: Structuring Pulumi Projects"
+date: 2023-03-16T16:50:38-07:00
+meta_desc: The third post in the Zephyr series examines recommended practices around structuring Pulumi projects.
+meta_image: meta.png
+authors:
+    - scott-lowe
+    - christian-nunciato
+    - aaron-kao
+tags:
+    - best-practices
+    - cloud-engineering
+    - zephyr
+---
+
+This is the third post in a series of blog posts focused on Zephyr Archaeotech Emporium---our fictional company---and their use of Pulumi to manage their online retail store. In the first post, you saw how Zephyr initially decided to go with a single Pulumi project for managing deployments of their online retail store application. In this post, you'll see how Zephyr's use of Pulumi evolves as their company grows and evolves.<!--more-->
+
+The [first blog post](/blog/iac-recommended-practices-code-organization-and-stacks/) in the Zephyr series specifically mentions that the series will show "how Zephyr's use of Pulumi evolves as the company grows and their online retail store application changes to accommodate their growth." While the ultimate goal of this series is to discuss recommended practices for using Pulumi to manage a fairly complex containerized application, it's important to note that many recommended practices are "point in time" recommendations---and they will change over time as the companies using Pulumi also change. This blog post is an example of that evolution in action: what was right for Zephyr while they were smaller may not be right for Zephyr when they are larger.
```

Comment:
```suggestion
The [first blog post](/blog/iac-recommended-practices-code-organization-and-stacks/) in the Zephyr series specifically mentions that the series will show "how Zephyr's use of Pulumi evolves as the company grows and their online retail store application changes to accommodate their growth." While the ultimate goal of this series is to discuss recommended practices for using Pulumi to manage a fairly complex containerized application, it's important to note that many recommended practices are "point in time" recommendations: best practices are context-dependent and the recommended practices for a given organization will change as the company and its infrastructure needs also change. This blog post is an example of that evolution in action: the practices that worked well for Zephyr while they were smaller and had smaller infrastructure and less teams working on that infrastructure are no longer ideal as the company has grown.
```

Reply:
General gist of the above suggestion is to avoid "they" "those" etc and explicitly restate the term for clarity. I think this is called getting rid of unqualified articles or something like that. I learned it in a college technical writing class and like the way it reads. YMMV.